src/common/options.cc

```diff
@@ -3380,6 +3380,22 @@ std::vector<Option> get_global_options() {
     .set_default(30)
     .set_description(""),
 
+    Option("osd_delete_sleep", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(0)
+    .set_description("Time in seconds to sleep before next removal transaction (overrides values below)"),
+
+    Option("osd_delete_sleep_hdd", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(5)
```

Comment:
5 seconds for HDD?

Reply:
these defaults seem ok to me. for e.g. 4mb objects, that's a limit of 24mb/s of deletes with the default osd_target_transaction_size of 30 objects, which seems reasonable for HDD.